Can not build admin CMS due to core.js issue

| Q | A
| --- | ---
| Sulu Version | 2.3.0 (also reproducible in 2.4.20) 
| PHP Version |  7.4.33 
| DB Version |  - 
| Browser Version |  - 

# Actual Behavior

 SULU interface doesn't correctly build

# Expected Behavior

 SULU interface to build and load successfully

# Steps to Reproduce

Run npm i and npm run build inside CLI 


# Possible Solutions


Downgrading core-js to 3.18.0 as per babel.config.json does not solve the issue
Currently using:
Node v14.21.3
NPM 6.14.18


Logs for version 2.4.20:
```
Build administration interface assets
-------------------------------------


> sulu-skeleton@ build /assets/admin
> webpack --mode production

   1406 modules

WARNING in ./node_modules/sulu-admin-bundle/services/ResourceRequester/registries/resourceRouteRegistry.js 28:4-18
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-admin-bundle/services/ResourceRequester/registries/resourceRouteRegistry.js 41:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-admin-bundle/services/ResourceRequester/registries/resourceRouteRegistry.js 51:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-media-bundle/containers/ImageMap/ImageRenderer.js 97:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-admin-bundle/stores/metadataStore/metadataStore.js 27:18-32
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-admin-bundle/containers/Badge/stores/BadgeStore.js 88:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in ./node_modules/sulu-admin-bundle/views/List/toolbarActions/UploadToolbarAction.js 154:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

ERROR in /vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router.js
Module not found: Error: Can't resolve 'core-js/modules/es.array.push.js' in '/Users/andreas/Workdata/DWBN-IT/sulu/sulu/vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js'

ERROR in /vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router.js
Module not found: Error: Can't resolve 'core-js/modules/esnext.iterator.constructor.js' in '/Users/andreas/Workdata/DWBN-IT/sulu/sulu/vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js'

ERROR in /vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js/router.js
Module not found: Error: Can't resolve 'core-js/modules/esnext.iterator.for-each.js' in '/Users/andreas/Workdata/DWBN-IT/sulu/sulu/vendor/friendsofsymfony/jsrouting-bundle/Resources/public/js'

[ERROR] Unexpected error while building administration interface assets.
```

Logs for version 2.3.0:
```
WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/registries/resourceRouteRegistry.js 29:4-18
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/registries/resourceRouteRegistry.js 41:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/registries/resourceRouteRegistry.js 50:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'

WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/stores/metadataStore/metadataStore.js 25:18-32
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'
 @ /var/www/html/application/vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/stores/metadataStore/index.js

WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Badge/stores/BadgeStore.js 87:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'
 
WARNING in /vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/UploadToolbarAction.js 153:11-25
"export 'default' (imported as 'symfonyRouting') was not found in 'fos-jsrouting/router'
```

The build command succeeds in 2.3.0, however the interface don't load and have the following errors in the console
![image](https://github.com/user-attachments/assets/560e6f3e-a98b-4da2-a73c-7f3c10e1aea9)

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -86,7 +86,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     test: /\.js$/,
                     // eslint-disable-next-line max-len
                     exclude: [/node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
-                        /friendsofsymfony\/jsrouting-bundle/
+                        /friendsofsymfony\/jsrouting-bundle/,
                     ],
                     use: {
                         loader: 'babel-loader',

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -87,7 +87,6 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     // eslint-disable-next-line max-len
                     exclude: [
                         /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
-                        /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
                         /friendsofsymfony\/jsrouting-bundle/,
                     ],
                     use: {

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -85,7 +85,9 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                 {
                     test: /\.js$/,
                     // eslint-disable-next-line max-len
-                    exclude: /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                    exclude: [/node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                        /friendsofsymfony\/jsrouting-bundle/
+                    ],
                     use: {
                         loader: 'babel-loader',
                         options: {

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -87,6 +87,7 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                     // eslint-disable-next-line max-len
                     exclude: [
                         /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                        /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
                         /friendsofsymfony\/jsrouting-bundle/,
                     ],
                     use: {

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -85,7 +85,8 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                 {
                     test: /\.js$/,
                     // eslint-disable-next-line max-len
-                    exclude: [/node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                    exclude: [
+                        /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
                         /friendsofsymfony\/jsrouting-bundle/,
                     ],
                     use: {

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -83,9 +83,11 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
         module: {
             rules: [
                 {
-                    test: /\.js$/,
-                    // eslint-disable-next-line max-len
-                    exclude: [/node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                    test: /\.js$/,                    
+                    exclude: [
+                        // eslint-disable-next-line max-len
+                        /node_modules[/\\](?!(sulu-(.*)-bundle|@ckeditor|ckeditor5|array-move|htmlparser2|lodash-es|@react-leaflet|react-leaflet)[/\\])/,
+                        // eslint-disable-next-line max-len
                         /friendsofsymfony\/jsrouting-bundle/,
                     ],
                     use: {

